docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -170,20 +128,16 @@ WARNING: Session Status iframe functionality is limited in some modern browsers.
 [[_javascript_implicit_flow]]
 ==== Implicit and hybrid flow
 
-By default, the JavaScript adapter uses the https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code] flow.
+By default, the adapter uses the https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code] flow.
 
-With this flow the {project_name} server returns an authorization code, not an authentication token, to the application. The JavaScript adapter exchanges
-the `code` for an access token and a refresh token after the browser is redirected back to the application.
+With this flow the {project_name} server returns an authorization code, not an authentication token, to the application. The JavaScript adapter exchanges the `code` for an access token and a refresh token after the browser is redirected back to the application.
 
-{project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth[Implicit] flow where an access token
-is sent immediately after successful authentication with {project_name}. This may have better performance than standard flow, as there is no additional
-request to exchange the code for tokens, but it has implications when the access token expires.
+{project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth[Implicit] flow where an access token is sent immediately after successful authentication with {project_name}. This may have better performance than standard flow, as there is no additional request to exchange the code for tokens, but it has implications when the access token expires.
 
 However, sending the access token in the URL fragment can be a security vulnerability. For example the token could be leaked through web server logs and or
 browser history.
 
-To enable implicit flow, you need to enable the `Implicit Flow Enabled` flag for the client in the {project_name} Admin Console.
-You also need to pass the parameter `flow` with value `implicit` to `init` method:
+To enable implicit flow, you need to enable the `Implicit Flow Enabled` flag for the client in the {project_name} Admin Console. You also need to pass the parameter `flow` with value `implicit` to `init` method:
```

Comment:
```suggestion
To enable implicit flow, you enable the *Implicit Flow Enabled* flag for the client in the {project_name} Admin Console. You also pass the parameter `flow` with the value `implicit` to `init` method:
```

Reply:
We eliminate "need to" or "have to" or "must" (when it follows you) because it adds no meaning and has a                    more demanding tone..  We just tell them to do something and that's it..